### PR TITLE
Refactored Create Accounts unit tests and added enum class & other class stubs

### PR DESCRIPTION
**Pull request by TerryvilleDev**
All the work I did from the first repo, along with some additional items

### CODE REVIEW

File: LearnDjango/tests/test_create_accounts.py [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+
+# Functions to setup all tests
+def all_tests_setup(test_user, id_number: int, name: str, password: str, email: str, address: str, phone: str):
+    test_user.user_id = id_number
+    test_user.name = name
+    test_user.password = password
+    test_user.email = email
+    test_user.home_address = address
+    test_user.phone_number = phone
```
Comment (bptome) [inline LearnDjango/tests/test_create_accounts.py:19]: When saving to the database, User.phone (in models) is an Integer field, so you may have to convert from String to Int, but I'm not sure if python has an explicit conversion

**Review by bptome — APPROVED** [review]
Looks good. I like the direct call to test_admin when checking duplicates. Just note the comment about using a String or Int for phone numbers otherwise your tests may not run. 

Note: I will probably add a toString in the enum because it will be needed for my stuff, just in case anyone was interested and so I remember to put it there lol.

File: LearnDjango/tests/test_create_accounts.py (hunk 1 above)
Comment (TerryvilleDev) [reply LearnDjango/tests/test_create_accounts.py:19]: I missed this.  The database diagram in the general channel defines it as a string.  I'll ask for clarification on the server and then make the changes needed.

**Review by OmarHHammad — APPROVED** [review]
LGTM

**Pull request merged**